packages/contracts/deploy/10_framework/99_verifiy_step.ts

```diff
@@ -39,6 +39,16 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
         `DAOENSSubdomainRegistrar isn't approved for all. Expected ${deployer} to have ${DAOENSSubdomainRegistrarAddress} approved for all`
       );
     }
+
+    const node = await DAOENSSubdomainRegistrar.node();
```

Comment:
Now, that I think about, it's really a bummer putting the `verify` such a below. 

The reason is: IF somehow it comes here, and fails, it means the previous steps(deployment of `daoregistry`, .e.t.c) are no longer applicable for us ! so gas were wasted.

not the current task though, just mentioned it, if we share the strong opinions of fixing this and moving each verify thing after each step. 

Though, the current way isn't too bad as well, at least running it on `rinkeby` gives you automatic confidence that it actually works and will definately work on mainnet too.

Reply:
not entirely. You can still manually change things for the deployment to progress. But this way we know that it works and everything is correct. Otherwise we need to do it at the end (after also deploying the plugins and registering them). This would waste even more gas